### DISCUSSION TITLE
new_term alert with constantly updating "seen values"

### DISCUSSION THREAD

**Discussion by ari-wein** [jertel/elastalert2#1090] [Q&A] [answered]
I would like to write an alert that triggers when a new term is seen, but only if it has not been seen within the last 5 days. It seems that the closest alert type for my use case would be the `new term`  type, but this doesn't seem to refresh the "seen values" every time an alert is triggered, only when Elastalert starts.

**Comment by jertel** [comment, marked as answer]
Can you share your rule parameters and some specific examples of how it's not working as expected?

**Reply by ari-wein** [reply]
I have not written the alert yet, but I will try to provide a little more context. I would like an alert that filters for a specific event, where every event has a username associated. The alert check all the usernames from all the events, and trigger an alert for usernames that have not appeared in any events in the last 5 days. this way, If a user is seen frequently, (at least once every 5 days) I should never get an alert. However, if the user hasn't been seen in the last 5 days, I would like an alert. This way, if a user appears less frequently, say, once a month, they will still trigger an alert.

**Reply by jertel** [reply]
That's a tough scenario for ElastAlert 2. You might have to schedule a restart every 5 days to get the "seen" list to repopulate. 

Perhaps another community user has a better solution for you.

**Reply by ari-wein** [reply]
Thanks for your help. I might be able to do this by alerting on alerts already in the alerts index. Is there an alerter that will just write to `/dev/null`, So I can get the "pre alerts" into the elastalert index without sending them anywhere?

**Reply by jertel** [reply]
You can use the _debug_ alerter:
```
alert: 
- debug
```
